ansible/configs/ocp4-cluster/pre_infra.yml

```diff
@@ -1,33 +1,49 @@
-- name: Step 000 Pre Infrastructure
+- name: Step 000 - Pre Infrastructure
   hosts: localhost
   connection: local
   become: false
   tags:
   - step001
   - pre_infrastructure
   tasks:
-    - debug:
-        msg: "Step 000 Pre Infrastructure"
-  tasks:
-    - import_role:
-        name: infra-local-create-ssh_key
-      when: set_env_authorized_key | bool
-      tags:
-        - generate_env_keys
-    - name: Ensure variables are set
-      assert:
-        that: "{{ item.0 }}"
-        msg: "{{ item.1 }}"
-      loop:
-        - - ocp4_pull_secret is defined
-          - ocp4_pull_secret variable must be defined
-    - name: if windows_password is not defined, generate one
-      when: windows_password is not defined
-      block:
-      - name: Generate windows Administrator password if not already defined
-        command: openssl rand -base64 25
-        register: password_gen_r
-      - name: set_fact generated_windows_password (just generated)
-        set_fact:
-          generated_windows_password: "{{ password_gen_r.stdout }}"
+  - debug:
+      msg: "Step 000 Pre Infrastructure"
+
+  - import_role:
+      name: infra-local-create-ssh_key
+    when: set_env_authorized_key | bool
+    tags:
+      - generate_env_keys
+
+  - name: Ensure variables are set
+    assert:
+      that: "{{ item.0 }}"
+      msg: "{{ item.1 }}"
+    loop:
+      - - ocp4_pull_secret is defined
+        - ocp4_pull_secret variable must be defined
+  - name: if windows_password is not defined, generate one
+    when: windows_password is not defined
+    block:
+    - name: Generate windows Administrator password if not already defined
+      command: openssl rand -base64 25
+      register: password_gen_r
+    - name: set_fact generated_windows_password (just generated)
+      set_fact:
+        generated_windows_password: "{{ password_gen_r.stdout }}"
+
+  - name: Set up AWS Student User
```

Comment:
What exactly is this used for?

Reply:
Azure I believe.

Reply:
`Set up aWS Student User` is used for Azure? That doesn't make any sense.

Reply:
Yeah. Actually it happens on EC2 as well if there are Windows VMs... May not be relevant for this config.

Reply:
Windows VMs? I don't see a conditional for that?